crates/ubrn_cli/src/commands/building.rs

```diff
@@ -58,6 +64,7 @@ impl BuildCmd {
         let mut files = match self {
             Self::Android(a) => a.build()?,
             Self::Ios(a) => a.build()?,
+            Self::Web(a) => a.build()?,
```

Comment:
does this code need wasm cfg/clap stuff as well, won't it be undefined if the wasm feature is missing?